tests/back/Pim/Enrichment/EndToEnd/Product/Product/ExternalApi/CreateProductEndToEnd.php

```diff
@@ -994,6 +995,32 @@ public function testResponseWhenAssociatingToNonExistingProductModel()
         $this->assertSame(Response::HTTP_UNPROCESSABLE_ENTITY, $response->getStatusCode());
     }
 
+    public function testAccessDeniedWhenCreatingProductWithoutTheAcl()
+    {
+        $client = $this->createAuthenticatedClient();
+        $this->removeAclFromRole('action:pim_api_product_edit');
+
+        $data =
+            <<<JSON
+    {
+        "identifier": "foo"
+    }
+JSON;
```

Comment:
```suggestion
        $data =
            <<<JSON
              {
                  "identifier": "foo"
              }
          JSON;
```
I thinks it's more readable to indent heredoc strings (non blocker just a heads up)

Reply:
I've did my best to stay consistent with other tests, which do not have the indentation.